Library/Homebrew/cask/lib/hbc/artifact/moved.rb

```diff
@@ -71,7 +71,7 @@ def move_back(skip: false, force: false, command: nil, **options)
           command.run("/bin/mv", args: [target, source], sudo: true)
         end
 
-        add_altname_metadata(target, source.basename, command: command)
+        add_altname_metadata(source, target.basename, command: command)
```

Comment:
Remove this completely. We don‘t need to add this when moving it back, actually.